hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/StateManager.java

```diff
@@ -58,7 +58,7 @@ void removePipeline(HddsProtos.PipelineID pipelineIDProto)
    * @throws IOException
    */
   @Replicate
-  void updatePipelineState(HddsProtos.PipelineID pipelineIDProto,
+  HddsProtos.Pipeline updatePipelineState(HddsProtos.PipelineID pipelineIDProto,
```

Comment:
There is no need to return `HddsProtos.Pipeline` as a response to `updatePipelineState`.

Reply:
Updated.